### PR TITLE
42269 Remove D365 payload key configuration changes

### PR DESCRIPTION
**Pull request by vipin14904**
1. Removed D365PayloadKeyConfiguration class and added D365PayloadKeyConstant class for constant variable as per suggestion. 

2. Rename Topic name from "fss-filesPublished" to "fss-filesPublished-AvcsData" NotificationTypes.json

### CODE REVIEW

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Services/SubscriptionServiceTest.cs [hunk 1]
```diff
@@ -17,25 +17,18 @@ namespace UKHO.ExternalNotificationService.API.UnitTests.Services
     public class SubscriptionServiceTest
     {
```
Comment (usha13529) [inline UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Services/SubscriptionServiceTest.cs:18]: Remove unused Using & Sort Usings

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Validation/D365PayloadValidatorTest.cs [hunk 2]
```diff
@@ -13,21 +13,14 @@ namespace UKHO.ExternalNotificationService.API.UnitTests.Validation
     public class D365PayloadValidatorTest
     {
```
Comment (usha13529) [inline UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Validation/D365PayloadValidatorTest.cs:14]: Remove unused Using & Sort Usings

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Services/SubscriptionService.cs [hunk 3]
```diff
@@ -14,13 +14,10 @@ namespace UKHO.ExternalNotificationService.API.Services
     public class SubscriptionService : ISubscriptionService
     {
```
Comment (usha13529) [inline UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Services/SubscriptionService.cs:15]: Remove unused Using & Sort Usings

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Validation/D365PayloadValidator.cs [hunk 4]
```diff
@@ -15,7 +15,7 @@ public interface ID365PayloadValidator
     }
     public class D365PayloadValidator : AbstractValidator<D365Payload>, ID365PayloadValidator
```
Comment (usha13529) [inline UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Validation/D365PayloadValidator.cs:16]: Remove unused Using & Sort Usings

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/Configuration/D365PayloadKeyConstant.cs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿
+namespace UKHO.ExternalNotificationService.Common.Configuration
+{
+    public class D365PayloadKeyConstant
+    {
```
Comment (usha13529) [inline UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/Configuration/D365PayloadKeyConstant.cs:5]: Make class static

**Review by usha13529 — CHANGES_REQUESTED** [review]
Please fix minor review comments.

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Services/SubscriptionServiceTest.cs (hunk 1 above)
Comment (vipin14904) [reply UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Services/SubscriptionServiceTest.cs:17]: Changes are done as requested.

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Validation/D365PayloadValidatorTest.cs (hunk 2 above)
Comment (vipin14904) [reply UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Validation/D365PayloadValidatorTest.cs:12]: Changes are done as requested.

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Services/SubscriptionService.cs (hunk 3 above)
Comment (vipin14904) [reply UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Services/SubscriptionService.cs:14]: Changes are done as requested

File: UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Validation/D365PayloadValidator.cs (hunk 4 above)
Comment (vipin14904) [reply UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Validation/D365PayloadValidator.cs:15]: Changes are done as requested

**Review by usha13529 — APPROVED** [review]
LGTM

**Review by barrie-cooper — APPROVED** [review]
All good thanks

**Pull request merged**